### PR TITLE
Commit changes to login button colour

### PR DESCRIPTION
**Pull request by WilliamChan4**
This pull request is to commit @lkihuynh's changes to remove the underline from the login button and change it's colour to blue.

### CODE REVIEW

File: src/frontend/src/pages/login/login.tsx [hunk 1]
```diff
@@ -10,6 +10,7 @@ import {
 } from "@material-ui/core";
 
 export const Login: React.FC = () => {
+
```
Comment (WilliamChan4) [inline src/frontend/src/pages/login/login.tsx:13]: Remove new line.

File: src/frontend/src/pages/login/login.tsx [hunk 2]
```diff
@@ -26,8 +27,13 @@ export const Login: React.FC = () => {
                 </Grid>
                 <Grid item>
                     <Box p={3}>
-                        <Link to="/home">
-                            <Button variant="contained">Login</Button>
+                        <Link to="/home" style={{ textDecoration: 'none' }}>
+                            <Button
+                                variant="outlined"
+                                color="primary"
+                            >
```
Comment (WilliamChan4) [inline src/frontend/src/pages/login/login.tsx:34]: Put onto one line.

**Comment by WilliamChan4** [comment]
@soroushMosaddad99 Could you please review this pull request since it could actually be quite useful. It's from an old branch that @lkihuynh was working on, but she never created a pull request for this. I've left some comments for things that could be fixed. If you like, you can branch off, fix the issues yourself, then merge back into this branch. 

Thanks :)

**Comment by soroushMosaddad99** [comment]
sure, no problem.

**Comment by WilliamChan4** [comment]
@soroushMosaddad99 Actually, I've made the changes that I wanted to this branch and @lkihuynh has approved.

**Comment by soroushMosaddad99** [comment]
I noticed, all good.

**Comment by WilliamChan4** [comment]
> I noticed, all good.

Thanks anyway :)

**Pull request merged**